modules/AlphaWalletFoundation/AlphaWalletFoundation/Tokens/Logic/GetContractInteractions.swift

```diff
@@ -11,29 +11,27 @@ import SwiftyJSON
 public class GetContractInteractions {
     struct E: Error {}
 
-    private let queue: DispatchQueue = .global(qos: .utility)
-
     public func getErc20Interactions(walletAddress: AlphaWallet.Address, server: RPCServer, startBlock: Int? = nil) -> Promise<[TransactionInstance]> {
         firstly {
             .value(walletAddress)
-        }.then(on: queue, { [queue] walletAddress in
-            return functional.getErc20Interactions(walletAddress: walletAddress, server: server, startBlock: startBlock, queue: queue)
+        }.then(on: .global(), { walletAddress in
```

Comment:
Should this have it's own private queue? Why use `global()` instead?

Reply:
oh yes, might be missed something, remoing these changes, and left as it is